token-client/src/main/java/com/sap/cloud/security/xsuaa/client/OAuth2TokenResponse.java

```diff
@@ -24,6 +25,10 @@ public OAuth2TokenResponse(@Nullable String accessToken, long expiredInSeconds,
 		this.refreshToken = refreshToken;
 	}
 
+	public OAuth2TokenResponse(@Nullable String accessToken, long expiredInSeconds, @Nullable String refreshToken, String tokenType) {
```

Comment:
i would turn it around, existing constructor calls new one, tokenType can be defaulted to "bearer" (see also [here](https://docs.cloudfoundry.org/api/uaa/version/4.30.0/index.html#jwt-bearer-token-grant
)).